src/app/donation-start/donation-start-primary/donation-start-primary.component.html

```diff
@@ -0,0 +1,493 @@
+
+<!--  Todo - move all content of this column **EXCEPT the main <form>** out of this template and in to
+      donation-start-container.component, along with support code in the .component.ts file, tests etc. It should be
+      possible to move piece by piece from the top start or end of the div.
+    -->
```

Comment:
What's the status of this todo? Do we still want to do it after this PR is merged?